getting-started/quickstart-guides/android.md

```diff
@@ -0,0 +1,389 @@
+---
+description: >-
+  Real-time image resizing, automatic optimization, and file uploading in
+  Android using ImageKit.io.
+---
+
+# Android
+
+This is a quick start guide to show you how to integrate ImageKit in an Android application.
+
+## **Clone and run the tutorial app**
+
+For this tutorial, it is recommended to use the sample Android app, as shown below. If you already have an existing Android app, it is also possible to use that, which will be explained in the continuation of the sample app.
```

Comment:
This statement seems ambiguous. It is a bit unclear. The first part talks about a recommended sample app, but the second part about an existing app and its continuation might confuse readers.
 
```suggestion
For this tutorial, we'll primarily use a sample Android app provided below. If you have your own existing Android app, you can integrate ImageKit into it. Instructions for both scenarios are provided in the subsequent sections.
```

Reply:
Modified this paragraph to clear the ambiguity. Please check